module-types/Datacom/FAN_OLT-DM4610.yaml

```diff
@@ -0,0 +1,6 @@
+---
+manufacturer: Datacom
+model: DM4610-FAN
+part_number: 800.5096.xx
```

Comment:
Just want to double check, is this the right part number? Can you like to the product/datasheet listing it?

Reply:
According to [this datasheet](https://www.datacom.com.br/uploads/product/file/a4d6b18b9b0cc9b4ac5dccfa22678fdd.pdf), this is the correct definition